client/public/index.html

```diff
@@ -9,6 +9,11 @@
       name="description"
       content="Web site created using create-react-app"
     />
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link
+      href="https://fonts.googleapis.com/css2?family=Montserrat:ital,wght@0,100;0,200;0,300;0,400;0,500;0,600;0,700;0,800;0,900;1,100;1,200;1,300;1,400;1,500;1,600;1,700;1,800&display=swap"
+      rel="stylesheet">
```

Comment:
Maybe consider downloading google fonts and import them locally from your projects sources. Linking to google fonts may result in cookies being saved on the users browser without their consent and thus legal consequences may follow.